hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ha/OMNodeDetails.java

```diff
@@ -31,43 +30,36 @@
 /**
  * This class stores OM node details.
  */
-public final class OMNodeDetails {
-  private String omServiceId;
-  private String omNodeId;
-  private InetSocketAddress rpcAddress;
+public final class OMNodeDetails extends NodeDetails {
   private int rpcPort;
-  private int ratisPort;
-  private String httpAddress;
-  private String httpsAddress;
 
   /**
    * Constructs OMNodeDetails object.
    */
   private OMNodeDetails(String serviceId, String nodeId,
       InetSocketAddress rpcAddr, int rpcPort, int ratisPort,
       String httpAddress, String httpsAddress) {
-    this.omServiceId = serviceId;
-    this.omNodeId = nodeId;
-    this.rpcAddress = rpcAddr;
+    super(serviceId, nodeId, rpcAddr, ratisPort, httpAddress, httpsAddress);
     this.rpcPort = rpcPort;
```

Comment:
is there any specific reason to maintain rpc port specifically in OMNodeDetails? I guess, it can be common for both SCM and OM.

Reply:
I might not have a correct understanding: 
SCM does not need a single RPC port, it has three: datanode, client and block server.

That why this diverges: OM maintains its rpc port while SCM maintains three ports.

If this is not correct, I can make a change in a subsequence PR.